test/test_n3.py

```diff
@@ -226,5 +252,24 @@ def testEmptyPrefix(self):
             g2), 'Document with declared empty prefix must match default #'
 
 
+class TestRegularExpressions(unittest.TestCase):
+    def testExponents(self):
+        signs = ("", "+", "-")
+        mantissas = ("1", "1.", ".1",
+                     "12", "12.", "1.2", ".12",
+                     "123", "123.", "12.3", "1.23", ".123")
+        es = "eE"
+        exps = ("1", "12", "+1", "-1", "+12", "-12")
+        for parts in itertools.product(signs, mantissas, es, exps):
+            expstring = "".join(parts)
+            self.assertRegex(expstring, exponent_syntax)
```

Comment:
```suggestion
            self.assertIsNotNone(exponent_syntax.match(expstring))
```

Reply:
I used `assetTrue` and `assertFalse` which are effectively doing the same.